checklists/sap_checklist.en.json

```diff
@@ -600,7 +609,8 @@
             "guid": "ce7bb122-f7c9-45f0-9e15-4e3aa3592829",
             "severity": "High",
             "training": "https://learn.microsoft.com/azure/azure-resource-manager/management/azure-subscription-service-limits",
-            "link": "https://learn.microsoft.com/azure/quotas/quotas-overview"
+            "link": "https://learn.microsoft.com/azure/quotas/quotas-overview",
+            "graph": "QuotaResources | where type =~ "microsoft.compute/locations/usages" | where subscriptionId in~ ("<Subscription1>","<Subscription2>") | mv-expand json = properties.value limit 400 | extend usagevCPUs = json.currentValue, QuotaLimit = json['limit'], quotaName = tostring(json['name'].localizedValue) | extend usagePercent = toint(usagevCPUs)*100 / toint(QuotaLimit) |where quotaName =~ "Total Regional vCPUs" or quotaName =~ "Total Regional Low-priority vCPUs" |project subscriptionId,quotaName,usagevCPUs,QuotaLimit,usagePercent,location,['json'] | order by ['usagePercent'] desc"
```

Comment:
You need to replace the double quotes (`"`) inside of the query by single quotes (`'`), othewise the JSON breaks.